jobbergate-cli/jobbergate_cli/subapps/job_scripts/app.py

```diff
@@ -210,6 +212,25 @@ def render(
             """
         ),
     ),
+    cluster_name: Optional[str] = typer.Option(
```

Comment:
This could be covered on the changelog

Reply:
Added an entry to the changelog file documenting this.